station_data.py

```diff
@@ -4,8 +4,21 @@
 from StringIO import StringIO
 from numpy import genfromtxt
 import ast
+import re
```

Comment:
You have to install [requests|http://docs.python-requests.org/en/latest/user/install/]. I'm on Ubuntu, so `sudo pip install requests` works for me. On Windows I'd probably have to download an executable and install it locally too. 

requests is a well known library in Python, with lots of users, good code quality and few bugs. It should work well in the project, and in case we find any error, it shouldn't be very hard to find a workaround/solution. Plus, its license is permissive (Apache License).